helm/slurm-cluster/values.yaml

```diff
@@ -5,6 +5,15 @@ annotations: {}
 pause: false
 # Slurm cluster type. Can be now gpu or cpu
 clusterType: gpu
+# partitionConfiguration define partition configuration of slurm worker nodes
+# https://slurm.schedmd.com/slurm.conf.html#SECTION_PARTITION-CONFIGURATION
+partitionConfiguration:
+  # Could be default or custom
+  configType: "default"
+  # configuration as raw multi string started with PartitionName
+  # Example:
+  # PartitionName=low_priority Nodes=worker-[0-9] Default=YES MaxTime=INFINITE State=UP PriorityTier=1
```

Comment:
Why the default partition is called "low_priority"?

Reply:
Because in this example u have 2 partitions hi and low, and by default u will use low. Which names do u want see here?

Reply:
Because this is an example of the "default" config. I expect a name like "default" or "main" (we have "main" now while it's not configurable)

Reply:
But that's example of not default config) If configType - default doesn't matter what u will put on rawConfig. Ok I'll change comment here: `Example for custom ConfigType`